### PR TITLE
Fix/bold weight

### PR DESCRIPTION
**Pull request by lancetharper**
Fixes the safari issue of not rendering bold if you have the font installed.

### CODE REVIEW

File: src/less/themes/default/elements/typography.variables [hunk 1]
```diff
@@ -30,7 +30,7 @@
 @brand-font-primary-name:                          "HCo Sentinel";
 @brand-font:                                       @brand-font-primary-name, @font-family-sans-serif;
 
-@brand-font-secondary-name:                        "Roboto Condensed", sans-serif;
```
Comment (mhewson) [inline src/less/themes/default/elements/typography.variables:33]: I worry about this not having a fall back, I would like it to be "Roboto Condensed", "Helvetica nue", sans-serif
does that cause a problem?

Comment (lancetharper) [reply src/less/themes/default/elements/typography.variables:33]: It shouldn't, it's just a fall back. Should I add Arial too like how it is a few lines up (commented out)?

Comment (mhewson) [reply src/less/themes/default/elements/typography.variables:33]: yes sir 
Roboto condensed -> roboto -> helvetica -> arial -> sans-serif

**Comment by mhewson** [comment]
approve

[Approved with PullApprove](https://pullapprove.com/rei/rei-cedar/pull-request/195/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=rei/rei-cedar)

**Pull request merged**